lib/grape/dsl/routing.rb

```diff
@@ -77,9 +77,12 @@ def do_not_route_options!
           namespace_inheritable(:do_not_route_options, true)
         end
 
-        def mount(mounts)
+        def mount(mounts, with: {})
```

Comment:
Do we use named parameters elsewhere. I know we no longer support older Rubies, but if we don't maybe make this consistent as `opts = {}` and `opts[:with] || {}`?